<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTASearchReplaceDialog</class>
 <widget class="QDialog" name="GTASearchReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="Find">
      <attribute name="title">
       <string>Find</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="FindLE1"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Find Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="SearchUpCB1">
            <property name="text">
             <string>Search Up</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="RegExCB1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Regular Expression</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="parameterSearchRB">
            <property name="text">
             <string>Parameter Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="textSearchRB">
            <property name="text">
             <string>Text search</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="MatchCaseCB1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Match Case</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="FindNextPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Find Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FindAllPB1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Find All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FindInAllDocPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Find All In Open documents</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Replace">
      <attribute name="title">
       <string>Replace</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="FindLE2">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Replace with</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="ReplaceLE2">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Replace Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="SearchUpCB2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Search up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="MatchCaseCB2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Match case</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="RegExpCB2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Regular expression</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="SearchNextPB2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Find Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReplacePB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>replaces parameters only</string>
           </property>
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReplaceAllPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>replaces in all parameters</string>
           </property>
           <property name="text">
            <string>Replace All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReplaceInAllDocsPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Repalce All In Open Documents</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Equipment_replace">
      <attribute name="title">
       <string>Equipment</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Equipment</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace with</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>110</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Smart_research">
      <attribute name="title">
       <string>Smart research</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>11</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Find :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>21</y>
         <width>491</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Search_requirement">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>87</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Find requirement</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Search_tag">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>127</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Find tag</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Search_comment">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>167</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Find comment</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FindLE1</tabstop>
  <tabstop>SearchUpCB1</tabstop>
  <tabstop>MatchCaseCB1</tabstop>
  <tabstop>RegExCB1</tabstop>
  <tabstop>textSearchRB</tabstop>
  <tabstop>parameterSearchRB</tabstop>
  <tabstop>FindAllPB1</tabstop>
  <tabstop>FindLE2</tabstop>
  <tabstop>ReplaceLE2</tabstop>
  <tabstop>SearchUpCB2</tabstop>
  <tabstop>MatchCaseCB2</tabstop>
  <tabstop>RegExpCB2</tabstop>
  <tabstop>SearchNextPB2</tabstop>
  <tabstop>ReplacePB</tabstop>
  <tabstop>ReplaceAllPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
